### DISCUSSION TITLE
group-hover:opacity-100 not working on Safari

### DISCUSSION THREAD

**Discussion by edgarasben** [tailwindlabs/tailwindcss#8635] [Help] [answered]
**What version of Tailwind CSS are you using?**

For example: v3.1.1

**What build tool (or framework if it abstracts the build tool) are you using?**

tailwind.new

**What browser are you using?**

Safari Version 15.5 

**What operating system are you using?**

macOS

**Reproduction URL**

https://play.tailwindcss.com/pfZWHS4f0h

**Describe your issue**

`group-hover:opacity-100` doesn't work on Safari 15.5 . Works on Brave, Chrome and Firefox.

**Comment by adamwathan** [comment]
Hey! Can't reproduce this:

https://user-images.githubusercontent.com/4323180/173560454-0f07f546-a171-499a-813d-593c32131a50.mov

Either way unfortunately I don't think there's anything we could change in Tailwind that would fix this if this was for some reason not working in Safari, that sounds like a Safari bug to me 😕

You should see the same issue here with no Tailwind at all in theory since this is the same CSS:

https://jsfiddle.net/6zfg025k/1/

**Reply by edgarasben** [reply]
Thanks for your prompt reply Adam!

Strange I don't have the same issue with regular CSS:


https://user-images.githubusercontent.com/725324/173565620-1f67aa9e-1de2-43a2-a029-4f2039d2e7f7.mov

**Reply by edgarasben** [reply]
Here is the video from Tailwind Play:

https://user-images.githubusercontent.com/725324/173566868-011a669b-11dd-44f2-8fec-d3a902fb9a86.mov

**Comment by rohmann** [comment, marked as answer]
Oh how I wish Safari wasn't the new IE...

It's actually working for me in the same Safari version. Video using the your tailwind play URL:

<img width="1440" alt="Screen Shot 2022-06-14 at 8 42 44 AM" src="https://user-images.githubusercontent.com/1506848/173579797-71bd647a-4c2a-4876-89bd-4bf70e0d7536.png">

If it's working on the jsfiddle, but not on tailwind then it might be something in your browser not letting that iframe function 100%. I noticed you've got a few extensions active. Occasionally I've seen cases where this accentuates issues. Might be worth a quick test in private browsing or with them disabled temporarily. Maybe that's preventing the iframe from getting all the styles.

Another thing you can check

1. Go to "Utilities" under generated styles and you'll see the Tailwind output.
2. Inspect the element in developer tools to make sure that style is getting applied. You can use the **Hover** checkbox to simulate a hover style

<img width="1440" alt="Screen Shot 2022-06-14 at 8 42 44 AM" src="https://user-images.githubusercontent.com/1506848/173579988-bc13c3b0-aba4-4aaa-a07e-e0488b6787bc.png">

**Reply by edgarasben** [reply]
You are right!

JSON Peep add-on was causing this... Problem resolves when I disable it. "Smooth and simple. Nothing more." — their slogan doesn't hold...

Thanks! :D

**Reply by rohmann** [reply]
Ok great! Glad to hear that's sorted out.
